profile/README.md

```diff
@@ -8,15 +8,15 @@
 
 </div>
 
-[ico-twitter]: https://img.shields.io/twitter/url?color=black&label=0xpolygonid&logoColor=black&style=social&url=https%3A%2F%2Ftwitter.com%2F0xpolygonid
+[ico-twitter]: https://img.shields.io/twitter/url?color=black&label=0xpolygonid&logoColor=black&style=social&url=https%3A%2F%2Ftwitter.com%2FPrivadoID
```

Comment:
@0xpulkit label for this button should be changed also to privadoID

Reply:
good catch @Lechu13 !

Reply:
Fixed it